core/templates/pages/splash-page/splash-page.component.spec.ts

```diff
@@ -37,10 +41,18 @@ describe('Splash Page', function() {
   beforeEach(angular.mock.module('oppia', function($provide) {
     $provide.value('WindowRef', windowRefMock);
   }));
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [HttpClientTestingModule]
+    });
+  });
+  beforeEach(angular.mock.module('oppia', function($provide) {
+    $provide.value('UserService', TestBed.get(UserService));
+  }));
```

Comment:
Instead of creating a new beforeEach block can we use the beforeEach on line 41 to provide `userService`?

Reply:
combined two beforeEach blocks